client/my-sites/plugins-wpcom/plugin-install-button.jsx

```diff
@@ -24,9 +26,15 @@ export const WpcomPluginInstallButton = props => {
 		siteId,
 		eligibilityData,
 		navigateTo,
-		initiateTransfer
+		initiateTransfer,
+		transferState
 	} = props;
 
+	const { COMPLETE } = transferStates;
+	if ( COMPLETE === transferState ) {
```

Comment:
this seems a bit redundant, though it's not wrong

if all we're doing though is using it once, why not use compare against `transferStates.COMPLETE`?